fuse/dl/models/backbones/backbone_resnet.py

```diff
@@ -47,9 +50,16 @@ def __init__(self, pretrained: bool = False, in_channels: int = 3, name: str = "
 
         # load pretrained parameters if required
         if pretrained:
+            print("Warning: not supported by new torchvision version - use weights instead")
+            from torch.hub import load_state_dict_from_url
```

Comment:
Do we want to keep support that option?

I suggest to use the `name` variable to pick the right wights in the new API ([API ref](https://pytorch.org/vision/stable/models.html) )

Reply:
Not really, for backward compatibility.
I agree - just wanted the cicd to pass for now. will open an issue.